warehouse/query-core/src/main/java/datawave/query/index/lookup/Union.java

```diff
@@ -208,7 +228,7 @@ private Tuple2<String,IndexInfo> advanceQueue() {
             
             IndexStream itr = children.poll();
             
-            pointers = pointers.union(itr.peek().second(), Lists.newArrayList(delayedNodes.getNodes()));
+            pointers = pointers.union(itr.peek().second(), Collections.emptyList());
```

Comment:
why are the delayed nodes dropped here?

Reply:
The delayed nodes are added at the very end. Otherwise the delayed terms are distributed into the union once per active index stream. I'll make a comment to that effect.